### PR TITLE
FORMS-192 # sub-forms optionally collapse with summary

### PR DESCRIPTION
**Pull request by jokeyrhyme**
- UX reviewed by Sam already

### CODE REVIEW

**Comment by jokeyrhyme** [comment]
I need to fix some add button duplication. Don't merge just yet. :)

**Comment by jokeyrhyme** [comment]
WIP! Don't merge this yet!

**Comment by jokeyrhyme** [comment]
Looks like FORMS-219 ( https://github.com/blinkmobile/forms/pull/17 ) fixed the double-rendering I thought I'd caused in this branch. Looks like there's nothing left to do here. :D

File: forms/jqm/subformcollapse.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    },
+
+    remove: function () {
+      this.$remove.off('click');
+
+      // don't need to unbind formElementEvents, because this is done in super
+      // events.proxyUnbindFormElementEvents(this, this.model, this.formElementEvents);
+
+      // using super's remove
+      return BMP.Forms._views.SubForm.prototype.remove.call(this);
```
Comment (simonmarklar) [inline forms/jqm/subformcollapse.js]: should you use  `SubFormView.prototype.remove.call(this)` here, incase `BMP.Forms._views.SubForm` has been overwritten ??

Comment (jokeyrhyme) [inline forms/jqm/subformcollapse.js]: Good call. I can't remember why I did it like this...

**Comment by simonmarklar** [comment]
:+1: LGTM

**Comment by ashishtilara** [comment]
LGTM :+1:

**Pull request merged**